examples/complete-vpn-gateway-with-logging/main.tf

```diff
@@ -0,0 +1,74 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "vpn_gateway" {
+  source = "../../"
+
+  vpn_gateway_id      = module.vpc.vgw_id
+  customer_gateway_id = aws_customer_gateway.main.id
+
+  vpc_id                       = module.vpc.vpc_id
+  vpc_subnet_route_table_ids   = module.vpc.private_route_table_ids
+  vpc_subnet_route_table_count = length(var.vpc_private_subnets)
+  local_ipv4_network_cidr      = "0.0.0.0/0"
+  remote_ipv4_network_cidr     = module.vpc.vpc_cidr_block
+
+  tunnel1_log_options = {
```

Comment:
Please add these two tunnel log options to the existing complete example. There is no need to have a separate example just for this.

Reply:
Done 👍